meteor/sofie-server/model/entities/rundown.ts

```diff
@@ -0,0 +1,213 @@
+import { Segment } from './segment'
+import { Part } from './part'
+import { Exception } from '../exceptions/exception'
+import { ErrorCode } from '../enums/error-code'
+import { LastPartInSegmentException } from '../exceptions/last-part-in-segment-exception'
+import { NotFoundException } from '../exceptions/not-found-exception'
+import { NotActivatedException } from '../exceptions/not-activated-exception'
+import { AlreadyActivatedException } from '../exceptions/already-activated-exception'
+import { AdLibPiece } from './ad-lib-piece'
+import { Piece } from './piece'
+
+export interface RundownInterface {
+	id: string
+	name: string
+	segments: Segment[]
+	isActive: boolean
+}
+
+export class Rundown {
```

Comment:
Apart from one property this again is like my comment on AdLibPiece.

Why does the property differ between the class and interface?

Reply:
No particular reason except for it being an oversight on my part.
I agree `isActive` is better